teams/DEPO/meeting-notes/incident-mitigation-planning.md

```diff
@@ -30,17 +30,17 @@ But the long term work won't bear fruit for another 6-12 months. So what can we
 
 Epic: [#22603](https://github.com/department-of-veterans-affairs/va.gov-team/issues/22603)
 
-- **Faster FE and BE build time:** will make firedrills less painful when they happen, because fixes will get through faster. It'll also make staging changes render faster once master merges, so teams can use the full window to QA their work prior to the prod deploy. ETA mid April 2021.
+- **Faster FE and BE build time:** will make firedrills less painful when they happen, because fixes will get through faster. It'll also make staging changes render faster once main merges, so teams can use the full window to QA their work prior to the prod deploy. ETA mid April 2021.
```

Comment:
I think it might make sense to keep these as `master` in this file, because these are notes from a meeting in the past, so `master` is more accurate. Plus it's also referring to `vets-api` as well, in some instances